### PR TITLE
Upgrade Ghostlab.app to v2.0

### PR DESCRIPTION
**Pull request by saetia**

### CODE REVIEW

**Comment by saetia** [comment]
what is expected here? i tried pushing a version with and without an empty line at the end, and i get the same "trailing whitespace detected". as far as i can tell my copy matches other cask files

File: Casks/ghostlab.rb [hunk 1]
```diff
@@ -1,12 +1,10 @@
 cask :v1 => 'ghostlab' do
   version :latest
   sha256 :no_check
-
```
Comment (vitorgalvao) [inline Casks/ghostlab.rb]: Don’t remove this empty line.

File: Casks/ghostlab.rb [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-  url 'http://awesome.vanamco.com/downloads/ghostlab/Ghostlab.dmg'
-  appcast 'http://awesome.vanamco.com/update/ghostlab-cast.xml'
+  url 'http://awesome.vanamco.com/Ghostlab2/downloads/Ghostlab2.dmg'
+  appcast 'http://awesome.vanamco.com/Ghostlab2/update/ghostlab2-cast.xml'
   name 'Ghostlab'
   homepage 'http://vanamco.com/ghostlab/'
   license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
-
```
Comment (vitorgalvao) [inline Casks/ghostlab.rb]: Nor this one.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by ndr-qef** [comment]
Thanks.

**Pull request merged**